honeybee_radiance_command/options/rcollate.py

```diff
@@ -0,0 +1,208 @@
+"""Rcollate parameters."""
+
+from .optionbase import OptionCollection, StringOptionJoined, StringOption, \
+    IntegerOption, BoolOption
+
+
+class RcollateOptions(OptionCollection):
+    """Rcollate command options.
+
+    [ -h[io] ][ -w ][ -f[afdb][N] ][ -t ][ -ic in_col ][ -ir in_row ][ -oc out_col ]
+    [ -or out_row ][ -o RxC[xR1xC2..] ]
+
+    In this class the -f option has been implemented such that there are separate options
+    like -fa , -ff etc. This is to enable the inclusion of numeric inputs for digits
+    in an easy manner.
+
+    Also see: https://www.radiance-online.org/learning/documentation/manual-pages/pdfs/rcollate.pdf
+    """
+    __slots__ = ('_h', '_w', '_fa', '_ff', '_fd', '_fb', '_t', '_ic', '_ir', '_oc',
+                 '_oR', '_o')
+
+    def __init__(self):
+        """rcollate command options"""
+        OptionCollection.__init__(self)
+        self._on_setattr_check = False
+        self._h = StringOptionJoined('h', 'Header availability.',
+                                     valid_values=('', 'i', 'o'))
+        self._w = BoolOption('w', 'Turn off non-fatal warning messages')
+        self._fa = StringOptionJoined('fa', 'Number of space-separated words/numbers in '
+                                            ' each record assumed to be in ASCII format.')
+        self._ff = StringOptionJoined('ff', 'Number of words in the input,'
+                                            ' each record assumed to be in '
+                                            'floating-point format.')
+        self._fd = StringOptionJoined('fd', 'Number of words in the input,'
+                                            ' each record assumed to be in double '
+                                            'format.')
+        self._fb = StringOptionJoined('fb', 'Number of words in the input,'
+                                            ' each record assumed to be in binary '
+                                            'format.')
+        self._t = BoolOption('t', 'Swap rows and columns in the input.')
+        self._ic = IntegerOption('ic', 'Input columns')
+        self._ir = IntegerOption('ir', 'Input rows')
+        self._oc = IntegerOption('oc', 'Output columns')
+        # or  is a protected keyword, so using oR
+        self._oR = IntegerOption('or', 'Output rows')
```

Comment:
We had cases like this before and we used `_` in those cases. We might want to do the same here to be consistent. Here is one for `as`:

https://github.com/ladybug-tools/honeybee-radiance-command/blob/e8f7459a6d36f6cd4a9df2ffbfd5ee9441188517/honeybee_radiance_command/options/rtrace.py#L709-L720